pandasai/helpers/save_chart.py

```diff
@@ -64,9 +67,13 @@ def add_save_chart(code: str, folder_name: str, print_save_dir: bool = True) ->
 
     """
 
-    # define chart save directory
-    project_root = dirname(dirname(dirname(__file__)))
-    chart_save_dir = os.path.join(project_root, "exports", "charts", folder_name)
+    if save_charts_path is not None:
+        charts_root_dir = save_charts_path
```

Comment:
I think some validation here to make sure `save_charts_path` is a valid path would be good. What do you think?

Reply:
I understand that it may be necessary to verify if the provided path is valid. However, I proceeded with the assumption that it is a string. Even if the path does not exist, the code will still generate the location.

One thing is notable, the previous implementation base path is the Path of File, Which seems not optimal.